README.md

```diff
@@ -75,6 +90,60 @@ when it reaches articles that are after the earliest accepted date, it will actu
  'url': 'https://www.iol.co.za/personal-finance/stanlib-may-further-reduce-fund-offering-14717297'}
 ```
 
+### Make a spider
+
+First, add this repository as a remote and make sure your local master is up to date:
+```bash
+git remote add upstream git@github.com:public-people/scrape-news.git
+git fetch upstream
+git checkout master
+git merge upstream/master
+```
+
+Create and check out a branch for the spider you're making:
+```bash
+git checkout -b newssite
+```
+
+Copy a spider from the repository and amend it as necessary. When it's done, ```git add``` it and ```commit``` the change to your working branch.
+
+
+To test individual xpaths you can use the scrapy shell:
+```bash
+scrapy shell "https://www.newssite.co.za/article-url"
+```
+If you go to the same url in your browser and right-click on, say, the title of the article, and select 'Inspect Element (Q)', you'll see something like
+```
+<h1 class="article-title">Title of article</h1>
+```
+highlighted. In the scrapy shell you can then enter something like
+```bash
```

Comment:
changing `bash` to `python` might give better syntax highlighting here

Reply:
but we're in the scrapy shell here, so wouldn't bash be more accurate?
I'll add something about then copying the tested xpath into the the spider, just realised I didn't cover that part.